src/OrchardCore.Themes/TheTheme/ResourceManifest.cs

```diff
@@ -0,0 +1,18 @@
+using OrchardCore.ResourceManagement;
+
+namespace OrchardCore.Themes.TheTheme
+{
+    public class ResourceManifest : IResourceManifestProvider
+    {
+        public void BuildManifests(IResourceManifestBuilder builder)
+        {
+            var manifest = builder.Add();
+
+            manifest
+                .DefineStyle("bootstrap-oc")
+                .SetUrl("~/TheTheme/css/bootstrap-oc.min.css", "~/TheTheme/css/bootstrap-oc.css")
```

Comment:
No more cdn?

Reply:
This is a custom local file. There is no cdn for it. It's the same reasoning as the local images for the theme.